stl/inc/xutility

```diff
@@ -126,6 +126,12 @@ void _Construct_in_place(_Ty& _Obj, _Types&&... _Args) noexcept(is_nothrow_const
         _Ty(_STD forward<_Types>(_Args)...);
 }
 
+// FUNCTION TEMPLATE _Construct_in_place_for_overwrite
+template <class _Ty>
+void _Construct_in_place_for_overwrite(_Ty& _Obj) noexcept(is_nothrow_default_constructible_v<_Ty>) {

```

Comment:
I really do not like the name, maybe `_Default_construct_in_place`?

If you do change it, would you mind using it for the `uninitialized_default_construct{_n}` family

Reply:
Done